mock_db.sh

```diff
@@ -0,0 +1,26 @@
+# This script handles local database creation to make testing easier.
+# Run with `make db`
+
+postgres -V
+if [ "$?" -gt "0" ]; then
+  echo "MAKE: Postgres not installed!"
+  exit 1
+fi
+
+echo "MAKE: Killing existing postgres processes..."
+pg_ctl -D /usr/local/var/postgres stop -s -m fast
+pg_ctl -D /usr/local/var/postgres start
+while true; do
+  pg_ctl -D /usr/local/var/postgres status
+  if [ "$?" -gt "0" ]; then
+    echo "MAKE: Waiting for postgres to start..."
+    sleep 3
+  else
+    break
+  fi
+done
+
+createuser -s rocket_test
+createdb rocket_test_db
+psql -d rocket_test_db -a -f ./schema/tables.sql
```

Comment:
If this were containerized you could take care of some of this by simply copying the schema to the right location in the Dockerfile:

```
COPY schema.sql /docker-entrypoint-initdb.d/
```

Reply:
tbh I've found Docker sucks even more for testing - starting anything is much slower than a simple pg instance, but I'll put this in docker